### PR TITLE
Add erc725js w/ npm auto-update

### PR DESCRIPTION
**Pull request by CJ42**
https://www.npmjs.com/package/@erc725/erc725.js

### CODE REVIEW

**Review by magalimorin18 — APPROVED** [review]
good idea

File: packages/e/erc725js.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ERC725Alliance/erc725.js/tree/develop"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@erc725/erc725.js",
+    "fileMap": [
+      {
+        "basePath": "",
```
Comment (MattIPv4) [inline packages/e/erc725js.json:22]:
```suggestion
        "basePath": "build/main/src",
```
The files under `build/main/src` make use of Node.js `require` calls and will not work in a browser. As such, I suspect you just want the files from `build/module/src`, which should work in browsers that support ESM.

File: packages/e/erc725js.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        "files": [
+          "**/*.@(js|ts)"
```
Comment (MattIPv4) [inline packages/e/erc725js.json:24]:
```suggestion
          "**/*.@(js|d.ts)"
```

File: packages/e/erc725js.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "erc725.js",
+  "filename": "erc725.min.js",
```
Comment (MattIPv4) [inline packages/e/erc725js.json:3]:
```suggestion
  "filename": "index.min.js",
```
It looks like the entrypoint file for the library is `index.js` (or `index.min.js` that we'll generate), not `erc725.min.js`

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/e/erc725js.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ERC725Alliance/erc725.js/tree/develop"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@erc725/erc725.js",
+    "fileMap": [
+      {
+        "basePath": "build/main/src",
```
Comment (MattIPv4) [inline packages/e/erc725js.json:22]:
```suggestion
        "basePath": "build/module/src",
```
The `main` build appears to make use of Node.js `require` calls that will not work in a browser. The `module` version appears to use `import` statements which should work for folks using ESM in the browser.

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**